graphqlapi/src/main/java/io/stargate/graphql/schema/schemafirst/migration/CassandraSchemaHelper.java

```diff
@@ -109,12 +115,63 @@ private static boolean equals(Column.ColumnType expectedType, Column.ColumnType
     return true;
   }
 
-  /** @return a list of differences, or empty if the UDTs match. */
-  public static List<Difference> compare(UserDefinedType expectedType, UserDefinedType actualType) {
+  private static SecondaryIndex findIndex(Table table, Column column) {
```

Comment:
any reason why not call it `findSecondayIndex()`?

Reply:
Not really, I'll rename it.